src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -235,6 +242,7 @@ def connection_update(cmd, client,
                       user_identity_auth_info=None, system_identity_auth_info=None,
                       service_principal_auth_info_secret=None,
                       key_vault_id=None,
+                      service_endpoint=None,
```

Comment:
This PR together with #21763 causes CI failure:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1464664&view=logs&j=36dd4138-4d53-5e46-00d9-e5cd9744be05&t=1cf3879c-0a42-5ccd-7693-7a4781d739d8&l=94

```
src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py:244:0: R0914: Too many local variables (26/25) (too-many-locals)
src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py:244:0: R0914: Too many local variables (26/25) (too-many-locals)
```